snap/snapcraft.yaml

```diff
@@ -110,36 +109,28 @@ apps:
   mysqld-exporter:
     command: start-mysqld-exporter.sh
     daemon: simple
+    install-mode: disable
     plugs:
       - network
       - network-bind
 parts:
   pin-mysql-apt:
     plugin: nil
     override-pull: |
-      printf "Package: mysql-server-8.0\nPin: version 8.0.32*\nPin-Priority: 1001\n" > /etc/apt/preferences.d/mysql
-      printf "\nPackage: mysql-router\nPin: version 8.0.32*\nPin-Priority: 1001\n" >> /etc/apt/preferences.d/mysql
-      printf "\nPackage: percona-xtrabackup-80\nPin: version 8.0.32*\nPin-Priority: 1001\n" >> /etc/apt/preferences.d/mysql
+      printf "Package: mysql-server-8.0\nPin: version 8.0.33*\nPin-Priority: 1001\n" > /etc/apt/preferences.d/mysql
+      printf "\nPackage: mysql-router\nPin: version 8.0.33*\nPin-Priority: 1001\n" >> /etc/apt/preferences.d/mysql
   packages-deb:
     plugin: nil
     stage-packages:
       - mysql-server-8.0
```

Comment:
Right now, we should ping versions here (is it technically possible here?):
```
mysql-server-8.0=8.0.33 
mysql-router=8.0.33
mysql-shell=8.0.33-0ubuntu0.22.04.1~ppa3
xtrabackup=8.0.33-27-0ubuntu0.22.04.1~ppa1
```
Sure, it will fail the next SNAP build after 8.0.34 release, but at least we will be informed about incompatible versions and prevent release by mistake. It is necessary until:
1) Security APT repo will reference several revisions
2) PPA packages moved into Ubuntu 24.04 LTS

Reply:
Pinning exact versions in [0fb3a58](https://github.com/canonical/charmed-mysql-snap/pull/35/commits/0fb3a5818649157a571a433acfeb1ee793b8a998)